test/na/test_nacomputation.cpp

```diff
@@ -39,6 +48,13 @@ TEST(NAComputation, Zone) {
   ss << zone;
   EXPECT_EQ(ss.str(), "zone");
 }
+TEST(NAComputation, ZonesExtent) {
+  const auto zone = Zone("zone", {0, 0, 2, 2});
+  EXPECT_TRUE(zone.contains({1., 1.}));
```

Comment:
Could be nice to have a convenience function that simply takes two doubles instead of a location for `contains`